hadoop-ozone/dist/src/main/smoketest/recon/recon-nssummary.robot

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 *** Settings ***
-Documentation       Smoke test for Recon Namespace Summary Endpoint for FSO buckets.
+Documentation       Smoke test for Recon Namespace Summary Endpoint for FSO and Legacy buckets.
```

Comment:
@dombizita Apart from the Namespace Summary for a directory there is no difference between FSO and LEGACY. In order to avoid duplication, we added the `BUCKET_LAYOUT` as a parameter and gave it the default value of FSO. The idea is to add an extra command to `run.sh` to also run the test with a parameter `-v BUCKET_LAYOUT:LEGACY`.